processes/rescale_chunks.json

```
@@ -0,0 +1,99 @@
+{
+  "namespace": "user",
+  "description": "It is a simple process to rescale Sentinel 2 RGB image along with the use of chunk_polygon  apply with a (User Defined Function) UDF.",
```

```suggestion
  "description": "It is a simple process to rescale Sentinel 2 RGB image along with the use of ``chunk_polygon()`` apply with a (User Defined Function) UDF.",
```

I don't understand this sentence. Are there words missing or so?

And you can link to other processes with the markup given above. :-)

---

modified